nvi-commons/src/test/java/no/sikt/nva/nvi/common/service/CandidateTest.java

```diff
@@ -126,6 +123,15 @@ public static Stream<Arguments> candidateResetCauseProvider() {
                                                                                List.of(new InstitutionPoints(
                                                                                    HARDCODED_INSTITUTION_ID,
                                                                                    randomBigDecimal(),
+                                                                                   null))))),
+                         Arguments.of(Named.of("affiliated institution changed",
```

Comment:
Argument that confirms bug

Reply:
Just to mention, there's `named("reason", thing)`